airbyte-cdk/python/airbyte_cdk/__init__.py

```diff
@@ -229,7 +228,7 @@
     "TypeTransformer",
     "ENV_REQUEST_CACHE_PATH",
     "create_timer",
-    "OneOfOptionConfig",
+    # "OneOfOptionConfig", TODO: this is a breaking change
```

Comment:
This will indeed impact a couple of connectors like:
* destination-milvus
* destination-weaviate
* source-google-drive
* source-sftp-bulk
* etc...